client/cli/tx.go

```diff
@@ -137,6 +137,18 @@ func TxRegisterAccountSignerlessly() *cobra.Command {
 				return err
 			}
 
+			// Since the signerless transaction does not require the AutoCCTP account to be in
+			// the keychain, all time a signerless tx is sent, the sequence is always zero. By
+			// querying the account it is possible to use the correct sequence.
```

Comment:
This reads better 👇🏻 

```suggestion
			// the keychain, by default, the transaction sequence is always zero. By
			// querying the account, it is possible to use the correct sequence.
```